standard/ERCs/erc-6900.md

```diff
@@ -82,16 +67,17 @@ Each step is modular, supporting different implementations, that allows for open
 
 - `IAccount.sol` from [ERC-4337](./eip-4337.md).
 - `IAccountExecute.sol` from [ERC-4337](./eip-4337.md).
-- `IModuleManager.sol` to support installing and uninstalling modules.
-- `IStandardExecutor.sol` to support open-ended execution. **Calls to modules through this SHOULD revert.**
+- `IModularAccount.sol` to support installing and uninstalling modules.
```

Comment:
This now also has `accountId`. We want this to be required and not optional, which is why that view function is not in IAccountLoupe.